lua/loc/fr.json

```diff
@@ -19,51 +21,72 @@
 	"bm_wp_upg_o_eotech_xps"    : "EOTech Modèle EXPS3",
 	"bm_wp_upg_o_reflex"        : "Viseur Générique Open Reflex",
-	"bm_wp_upg_o_rx01"          : "Viseur Trijicon RX01 Polarized Reflex",
+	"bm_wp_upg_o_rx01"          : "Viseur Trijicon RX01 Reflex polarisé",
 	"bm_wp_upg_o_rx30"          : "Viseur Trijicon RX30 Reflex",
   "bm_wp_upg_o_ak_scopemount" : "Monture optique K-VAR KV-045",
-  "bm_wp_fps_upg_o_45iron"    : "Monture d'angle XTI Angle",	
+  "bm_wp_fps_upg_o_45iron"    : "Monture d'angle XTI",
   "bm_wpn_fps_upg_o_45rds_v2" : "Monture d'angle Aimpoint Micro T-2",
 	"bm_wp_upg_o_box"           : "Viseur Pulsar Digisight LRF N960 N970 nocturne avec chercheur de cibles",
   "bm_wpg_o_mbus_rear"        : "Mires avant et arrière MBUS",
   "bm_wp_upg_o_rmr"           : "Viseur Trijicon RM06 RMR",
   "bm_wp_upg_o_spot"          : "Double Monture Optique avancée NcSTAR",
 
+	"bm_wp_m4_m_straight" : "Magasin 22 cartouches STANAG",
+	"bm_wp_m4_uupg_m_std" : "Magasin 38 cartouches STANAG",
+	"bm_wp_upg_m4_m_quad" : "Magasin 60 cartouches STANAG",
   "bm_wp_m4_m_quick"    : "Magpul 5.56 NATO Original",
 	"bm_wp_sr2_m_quick"   : "Magpul 9mm Veresk Retrofit Original",
-  
+
 	"bm_wp_upg_o_sho_salvo_large"  : "Suppresseur SilencerCo Salvo 12",
 	"bm_wp_upg_ns_sho_salvo_large" : "Suppresseur SilencerCo Salvo 12",
 	"bm_wp_upg_ns_shot_thick"      : "Suppresseur Chigurh",
 	"bm_wp_upg_ns_ass_smg_medium"  : "Suppresseur Customisé SAI M80",
 	"bm_wp_upg_ns_ass_smg_small"   : "Suppresseur GEMTECH HALO",
-	"bm_wp_upg_ns_ass_smg_firepig" : "Suppresseur de flammes Noveske KX3 Flash Suppressor",
+	"bm_wp_upg_ns_ass_smg_firepig" : "Suppresseur de flammes Noveske KX3",
+	"bm_wp_upg_ns_ass_smg_stubby"  : "Suppresseur de flammes VFC Stinger",
+	"bm_wp_upg_ns_ass_smg_large"   : "Gros Suppresseur générique",
 	"bm_wp_upg_ns_ass_pbs1"        : "Suppresseur Sonore PBS-1",
 	"bm_wp_upg_ns_medium_gem"      : "Suppresseur GEMTECH BLACKSIDE-45",
-	"bm_wp_upg_ns_large_kac"       : "Suppresseur Customisé Knight's Armament Mk23",
-	"bm_wp_upg_ns_pis_small"       : "Suppresseur Customisé Thompson Machine Poseidon",
+	"bm_wp_upg_ns_large_kac"       : "Suppresseur Knight's Armament Mk23",
+	"bm_wp_upg_ns_pis_small"       : "Suppresseur Thompson Machine Poseidon",
 	"bm_wp_upg_ns_pis_large"       : "Suppresseur SilencerCo Osprey .45",
 	"bm_wp_upg_ns_ass_filter"      : "Suppresseur Filtre à Huile",
 	"bm_wp_mosin_b_sniper"         : "Suppresseur Bramit",
+	"bm_wp_upg_ns_pis_jungle"      : "Suppresseur customisé M1911A1 camouflé",
+	"bm_wp_upg_ns_pis_medium_slim" : "Suppresseur chromé",
+	"bm_wp_sr2_ns_silencer"        : "Suppresseur 9x39mm",
 
 	"bm_wp_upg_ass_ns_jprifles" : "Compensateur JP Profil-standard",
 	"bm_wp_upg_ass_ns_linear"   : "Compensateur Linéaire KIES Blast Master",
 	"bm_wp_upg_ass_ns_surefire" : "Frein de Bouche SureFire MB556K",
+	"bm_wp_upg_ns_shot_shark"   : "Bouche Tromix Shark Ouvre porte",
```

Comment:
I've hesitated on that one, Because "Open door" has i've say it was pretty soft, but the other alternative was "Défonce porte" wish sounds a bit more vulgar, and a lot more brutal as the style of "Porte fucker" (The verb "Défoncer" is mostly beating something up to a extreme level)